doc/requirements.txt

```diff
@@ -1,6 +1,74 @@
-docutils==0.16
-sphinxcontrib-bibtex
-pygments-github-lexers
-cirq
-pennylane
-sphinx-automodapi
```

Comment:
I can make the docs locally, but for some reason, build the docs fails. Also some of the dependencies listed here are not required for me to build the docs locally ...